src/main/java/uk/gov/ons/fwmt/job_service/service/impl/JobServiceImpl.java

```diff
@@ -57,8 +58,10 @@ public SampleSummaryDTO processSampleFile(MultipartFile file)
     jobProcessor.processSampleFile(f);
     return sampleSummaryDTO;
```

Comment:
Since this is top level file:
Add log.info before calling sendCSV , before validation and before calling processSampleFile

Reply:
This has now been added.